### DISCUSSION TITLE
Intelbras/Dahua go2rtc integration solution

### DISCUSSION THREAD

**Discussion by eccentricexit** [blakeblackshear/frigate#12536] [Show and tell]
I've been bashing my head for a whole week trying to integrate my Intelbras VIPC 1230 B with go2rtc and frigate. I finally figured it out and thought I'd share since I could find no answer to this online.

Intelbras cameras are just rebranded Dahuas so it may help owners of those as well.

The issue was that my stream would play perfectly fine on the camera's web interface and on VLC, but if I put go2rtc in the middle it would not work properly.

The trick was to change RTSP transport from TCP to UDP+TCP with `#input=rtsp/udp`, so my fulll URL looked like this:

`ffmpeg:rtsp://user:password@192.168.55.81/cam/realmonitor?channel=1&subtype=0#input=rtsp/udp`

Cheers

**Comment by pbassut** [comment]
Thank you! Worked well for me!
